Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:stretch
+FROM debian:buster-20200607-slim
```

Comment:
Why fixed date base? Using `debian:buster-slim` would be more aligned with what we do for Agent7

Reply:
Reproducible builds. I personally think this is better. But if you have a strong opinion let me know, we can fix in a follow-up PR.